src/networking.c

```diff
@@ -714,6 +714,24 @@ void setDeferredAggregateLen(client *c, void *node, long length, char prefix) {
      * we return NULL in addReplyDeferredLen() */
     if (node == NULL) return;
 
+    /* Things like *2\r\n, %3\r\n or ~4\r\n are emitted very often by the protocol
+     * so we have a few shared objects to use if the integer is small
+     * like it is most of the times. */
+    const size_t hdr_len = length < 10 ? 4 : 5;
```

Comment:
so maybe we'll make a macro for this and declare it next to OBJ_SHARED_BULKHDR_LEN, so that if someone some day adds 128 of these, it'll be more likely that they'll update it.

Reply:
addressed in https://github.com/redis/redis/pull/10334/files#diff-e90a5ffcc23440393815655ee66ce9301d183a681056ee8020e39d5df2c4fa0fR112